cmd/antrea-agent/agent.go

```diff
@@ -63,15 +64,17 @@ func run(o *Options) error {
 	}
 
 	// Create ovsdb and openflow clients.
-	ovsdbConnection, err := ovsconfig.NewOVSDBConnectionUDS("")
+	ovsdbAddress := ovsconfig.GetConnAddress(o.config.OVSRunDir)
```

Comment:
Since all of ovsdbConnection, ovsBridgeMgmgAddr, and ovsBridge name are about OVS bridges, what about using a struct to provide the three methods, and the struct is constructed with o.config.OVSRunDir and o.config.OVSBridg. Then we could leverage the struct in later callers.

Reply:
Currently, we alreay have two bridge structs:
- "ovsconfig.OVSBridge": Need ovsdbAddress
- "openflow.OFBridge": Need ovsBridgeMgmgAddr

I think directly set a primitive address value to when init the struct is enough clear. This patch mainly focus on the OF connection on windows. Can we consider to refactor the struct if there're specific requirement?

Reply:
Sure.